README.md

```diff
@@ -166,9 +167,9 @@ toolchainsPlugin {
   registerDebugBuildType = true
 
   // Add the roborio compiler
-  withRoboRIO()
+  withCrossRoboRIO()
   // Add the raspbian compiler
-  withRasbian()
+  withLinuxArm32()
```

Comment:
It is, but technically the other linux platforms have the same issues. Also 32 bit arm is pretty much dead anyway, other then the pi.

Reply:
And the roboRIO, which is another 32-bit ARM platform running Linux.

Reply:
I can't really make it unambiguous. It doesn't work because the roborio is soft float. I could change it to arm32hf I guess.

Reply:
The actual architecture name has to be arm32 for gradle reasons.

Reply:
The way toolchains generally solve this is with triples that include the CPU arch, vendor sysroot name, and OS.

Reply:
I know it doesn't work for the roboRIO. My point is the name doesn't make that obvious.

Reply:
I can also just add NonRoboRIO to the name. TBH I don't really care, its an internal build setup thing not seen by teams.

Reply:
Actually, I'm just going to keep it as is. Because of gradle, the actual platform name has to stay linuxarm32. So its just going to have to be implied that roborio is special.

I'll add cross to all of them, to make it clear theyre cross compile setups.